app.js

```diff
@@ -4,6 +4,37 @@ import * as JobRunner from './lib/job-runner.js';
 import * as DownloadJob from './lib/download-job.js';
 import * as Delta from './lib/delta.js';
 
+/**
+ * application architecture:
+ * 1. frontend saves a PublicationMetricsExportJob in the database
+ * 2. delta-notifier sends a notification to this service
+ * 3. publication-report-service comes in action:
+ *    flow: (based on the file-bundling-service @see https://github.com/kanselarij-vlaanderen/file-bundling-service}
+ *    1. /delta endpoint picks up notification @see file://./lib/delta.js
+ *    2. JobRunner sets status to Running and runs DownloadJob @see file://./lib/job-runner.js
+ *    3. DownloadJob runs @see file://./lib/download-job.js
+ *      1. parse job parameters @see file://./job-params.js
+ *      2. build report query @see file://./queries/reports/index.js
+ *      3. request Virtuoso in order generate csv @see file://./lib/query-csv.js
+ *      4. save the file and database record @see file://./lib/mu-files.js
+ *    4. JobRunner sets status to Success/Fail
+ * 4. when frontend notices the job has finished, it shows the download link
+ */
+
+/**
+ * /// TOREVIEW:
+ * in order to interpret the dates (parsed from in .toJSON() format (=.toISOString())) in the same timezone as the frontend
+ *  node.js allows setting the timezone
+ *
+ * alternatives:
+ * configure docker:
+ * a timezone file for the docker container and mount it to /etc/timezone
+ * @see https://serverfault.com/questions/683605/docker-container-time-timezone-will-not-reflect-changes#
+ */
+if (!process.env.TZ) {
+  process.env.TZ = 'Europe/Brussels';
+}
```

Comment:
For me this looks like a good solution for now, at least in this PR.
@MikiDi for you as well?

Possibly in a different ticket: should we allow the timezone to be specified as a parameter with the request? Or is that needless effort since we always know which timezone our app is running in?

Reply:
If this is about tweaking the server timezone to properly interpret the client timestamp that doesn't contain an explicit offset, I suggest to add an explicit offset on the client side instead. Also see https://github.com/mu-semtech/mu-javascript-template/issues/40

Reply:
Het probleem is omgekeerd. De datums krijgen een offset mee in de frontend als volledige `DateTimes` (via toISOString). Ze vallen dus in de vorige dag in Greenwich-tijd. De publication-dates zijn in Virtuoso opgeslagen als 'date'. Logisch zou ik deze waarde dus moeten converteren naar een `date` in de SPARQL-query. Maar de server moet weten in welke tijdzone hij staat, om te bepalen welke dag het is, gegeven de dateTime-timestamp.

> For one-timezone userbase apps, there is no discussion on what "local" means

Ik dacht dat het de bedoeling was om onze applicatie in Belgische tijd te houden. Om het zoals simpel te houden moeten dan zo als ge zegt, frontend en services het eens zijn over wat lokaal is.

Reply:
Ik heb KAS-3445 hiervoor aangemaakt om dit op niveau van de gehele applicatie eens te bekijken, want ik geraak er ook niet meer aan uit ;-)